src/main.rs

```diff
@@ -96,6 +105,41 @@ async fn main() -> std::io::Result<()> {
         .new_game(GameOptions {}, Some("test".to_string()))
         .unwrap();
 
+    let reaper_manager_ref = game_manager_wrapper.clone();
+
+    // Game Reaper thread
+    thread::spawn(move || loop {
+        thread::sleep(Duration::from_secs(REAP_INTERVAL));
+        let test_handle = GameHandle("test".to_string());
+        let num_games = reaper_manager_ref
+            .game_manager
+            .read()
+            .unwrap()
+            .games
+            .len()
+            .clone();
+
+        let mut game_manager_mut = reaper_manager_ref.game_manager.write().unwrap();
+        let games = game_manager_mut.games.clone();
+        let mut unreaped_games: HashMap<GameHandle, std::sync::Arc<RwLock<GameWrapper>>> =
+            HashMap::new();
+        for (handle, game_wrapper) in games {
+            let creation_time = game_wrapper.read().unwrap().game.game_created;
+            if handle == test_handle || (creation_time + REAP_DURATION) > get_current_time_secs() {
+                unreaped_games.insert(handle, game_wrapper);
+            }
```

Comment:
It'd be good to have an else block saying the names of the games you _are_ reaping.

Reply:
wait, like something to log through each of the names of the reaped games? (seems legit to me)

Reply:
Ya that.